terraform/main.tf

```diff
@@ -126,6 +126,7 @@ variable "pubsub_topics" {
     "froge_of_the_day",
     "private_message_discord",
     "update_user_role",
+    "welcome_new_user",
```

Comment:
Delete this as well since we are gonna user the `channelMessage` function

Reply:
Done